docs/content/getting-started/create-new-project.mdx

```diff
@@ -7,20 +7,22 @@ description: Dagster comes with a convenient CLI command for creating a new proj
 
 The easiest way to start building a Dagster project is by using the `dagster project` CLI. This CLI tool helps generate files and folder structures that enable you to quickly get started with Dagster.
 
-You can scaffold a new project [using the default project skeleton](#option-1-starting-with-the-default-project-skeleton), or [start with using one of the official Dagster examples](#option-2-starting-with-an-official-example).
-
 ---
 
-## Option 1: Starting with the default project skeleton
+## Bootstrapping a new project
+
+You can scaffold a new project using the default project skeleton, or start with one of the official Dagster examples.
 
+<TabGroup>
+<TabItem name="Starting with the default project skeleton">
 To get started, you can run:
```

Comment:
It might be good to add an H3 here so users can jump to this section (and the official example tab) if they want to. Ex: `### Using the default project skeleton` and `### Using an official example` It'll also show up in the sidenav, too!